ops/prod/metabase.tf

```diff
@@ -3,6 +3,11 @@ module "metabase" {
   name   = "${local.project}-${local.name}-${local.env}-metabase"
   env    = local.env
 
+  app_settings_overrides = {
+    "MB_DB_USER" = "${data.azurerm_key_vault_secret.postgres_nophi_user.value}@${data.terraform_remote_state.persistent_prod.outputs.postgres_server_name}"
+    "MB_DB_PASS" = data.azurerm_key_vault_secret.postgres_nophi_password.value
```

Comment:
If we're just using this to pass in the username and password, might it make sense to have those as explicitly named inputs (one of which is sensitive, the other not) rather than plugging them directly into the metabase environment variables from here?

Reply:
I was trying to avoid more copy-pasta but technically yes, we should be passing in these variables

Reply:
Actually I misread this question. I think I would prefer not to obfuscate the variables like that, since if we did pass them in as named inputs we'd just have to do the env var thing inside the module instead.

Reply:
Fixed in new commit